source/scss/_pf-component-card.scss

```diff
@@ -2,7 +2,7 @@
 // Base styles
 //
 
-.card {
+.pf-card {
```

Comment:
all cards (including panels) have the shadow. So that one should actually be `.card`, sorry